### PR TITLE
Fix document

### PR DESCRIPTION
**Pull request by okuta**

### CODE REVIEW

File: docs/source/compatibility.rst [hunk 1]
```diff
@@ -1,26 +1,26 @@
...
 Note that this document may contain ambiguities on the level of supported compatibilities.
 
 
 Targeted Versions
 -----------------
 
-This policy is applied to Chainer of versions v1.5.1 and higher.
-Note that this policy is not applied to Chainer of lower versions.
+This policy is applied to CuPy of versions v1.0.0 and higher.
+Note that this policy is not applied to CuPy of lower versions.
```
Comment (unnonouno) [inline docs/source/compatibility.rst]: Is this line required?

File: docs/source/compatibility.rst [hunk 2]
```diff
@@ -85,15 +85,15 @@ Any API that is not experimental is called *stable* in this document.
...
 This change is not considered as a break of backward compatibility.
-Chainer can promote an experimental API to become stable at any **minor** or **major** version up.
+CuPy can promote an experimental API to become stable at any **minor** or **major** version up.
 Once experimental APIs become stable, they cannot revert to experimental again.
 
 When users use experimental APIs for the first time, warnings are raised once for each experimental API,
 unless users explicitly disable the emission of the warnings in advance.
 
-See the document of :meth:`chainer.utils.experimental` how developers mark APIs as experimental
+See the document of :meth:`cupy.utils.experimental` how developers mark APIs as experimental
```
Comment (unnonouno) [inline docs/source/compatibility.rst]: CuPy doesn't have experimental.

File: docs/source/contribution.rst [hunk 3]
```diff
@@ -235,11 +235,11 @@ In order to write tests for multiple GPUs, use ``cupy.testing.attr.multi_gpu()``
       def test_my_two_gpu_func(self):
           ...
 
-If your test requires too much time, add ``chainer.testing.attr.slow`` decorator.
+If your test requires too much time, add ``cupy.testing.attr.slow`` decorator.
```
Comment (unnonouno) [inline docs/source/contribution.rst]: CuPy doesn't have slow

File: docs/source/compatibility.rst (hunk 2 above)
Comment (okuta) [inline docs/source/compatibility.rst]: Yes, not yet.

File: docs/source/contribution.rst (hunk 3 above)
Comment (okuta) [inline docs/source/contribution.rst]: Yes, not yet.

File: docs/source/compatibility.rst (hunk 1 above)
Comment (okuta) [inline docs/source/compatibility.rst]: I will delete

**Comment by unnonouno** [comment]
LGTM

**Pull request merged**